### ISSUE TITLE
Storing player brush data between saves

### ISSUE THREAD

**Issue by Ghoulboy78** [Ghoulboy78/Scarpet-edit#104]
Just that. Also I would propose to add /brush data or /brush help which would tell the player what brushes they currently have configured.

**Comment by Ghoulboy78** [comment]
This extends to all player data

**Comment by Firigion** [comment]
`/brush list` already does that.

**Comment by Ghoulboy78** [comment]
True, `brush list` already tells what brushes you have configured.
But the first point still stands
